sdk-network-providers/api_network_provider.md

```diff
@@ -0,0 +1,60 @@
+## ApiNetworkProvider
+
+Sometimes the `ApiNetworkProvider` class uses an underlying `ProxyNetworkProvider`as some of the API routes are forwarded to the proxy.
+
+```
+class ApiNetworkProvider:
+    // The constructor is not captured by the specs; it's up to the implementing library to define it.
+    // Should specify the url to query and have a backing proxy provider
```

Comment:
Backing proxy provider is not necessary because:
1. in our case, API is forwarding unknown requests to his configured proxy
2. one could use an API without knowing the proxy

Other arguments can be: req timeout, HTTP headers ec.